jcg_evaluation/src/main/scala/EvaluationHelper.scala

```diff
@@ -2,57 +2,38 @@ import java.io.File
 
 // todo make a factory
 class CommonEvaluationConfig(
-        val DEBUG:                   Boolean,
-        val INPUT_DIR_PATH:          String,
-        val OUTPUT_DIR_PATH:         String,
-        val EVALUATION_ADAPTERS:     List[JavaTestAdapter],
-        val PROJECT_PREFIX_FILTER:   String,
-        val ALGORITHM_PREFIX_FILTER: String
-) {
+                              val DEBUG: Boolean,
```

Comment:
All of this indentation here doesn't look good.

Reply:
Hmmm, this might be a problem with my formatting settings for IntelliJ.
Which setings do or should we actually use? The ones in the project don't seem to match the formatting that has actually been used throughout the project

Reply:
Try the IntelliJ and Scalafmt configs from the OPAL project